cloudformation.yaml

```
@@ -3,46 +3,33 @@ AWSTemplateFormatVersion: "2010-09-09"
 Description: "Step Function for rotating Elasticsearch nodes"
 
 Parameters:
-  Stage:
-    Type: String
-    Description: Stage name.
   Stack:
     Type: String
-    Description: The name of the stack, for context in alerts and/or running node rotation against multiple clusters.
+    Description: Stack name defined in riff-raff.yaml to support per-account continuous deployment of this project. Not required if the template will be updated manually.
+  App:
+    Type: String
+    Description: App name defined in riff-raff.yaml to support per-account continuous deployment of this project
+    Default: elasticsearch-node-rotation
+  Stage:
+    Type: String
+    Description: Stage name for Riff-Raff deploys to support per-account continuous deployment of this project
+    Default: PROD
   DeployS3Bucket:
     Type: String
     Description: Bucket which contains .zip file used by lambda functions e.g. deploy-tools-dist.
   DeployS3Key:
     Type: String
-    Description: Key for .zip file used by lambda functions e.g. <stack>/<stage>/elasticsearch-node-rotation/elasticsearch-node-rotation.zip.
-  DataNodeAsg:
-    Type: String
-    Description: The name of the auto-scaling group which contains the Elasticsearch data nodes (which may also be masters).
-  WarmDataNodeAsg:
-    Type: String
-    Description: The name of the warm auto-scaling group which contains the Elasticsearch warm data nodes (optional).
-  DedicatedMasterAsg:
-    Type: String
-    Description: The name of the auto-scaling group which contains dedicated Elasticsearch master nodes. Leave this field empty if the Elasticsearch cluster does not have dedicated master nodes.
-  DataNodeRotationCronExpression:
+    Description: Key for .zip file used by lambda functions e.g. <stack>/<stage>/<app>/<app>.zip.
+  RotationCronExpression:
     Type: String
     Description: Cron expression which determines when data node rotations are scheduled.
```

Minor: I think this cron will be used for dedicated master nodes too?
```suggestion
    Description: Cron expression which determines when node rotations are scheduled.
```